evap/staff/tests/test_views.py

```diff
@@ -966,18 +969,116 @@ def test_operation_start_evaluation(self):
         self.assertEqual(evaluation.state, 'in_evaluation')
 
     def test_operation_prepare(self):
-        evaluation = baker.make(Evaluation, state='new', course=baker.make(Course, semester=self.semester))
+        evaluation = baker.make(Evaluation, state='new', course=self.course)
         urloptions = '?evaluation={}&target_state=prepared'.format(evaluation.pk)
 
         response = self.app.get(self.url + urloptions, user='manager')
         self.assertEqual(response.status_code, 200, 'url "{}" failed with user "manager"'.format(self.url))
-
         form = response.forms['evaluation-operation-form']
         form.submit()
 
         evaluation = Evaluation.objects.get(pk=evaluation.pk)
         self.assertEqual(evaluation.state, 'prepared')
 
+    def submit_operation_prepare_form(self, url_options):
```

Comment:
this method could return the emails directly instead of going through self.actual_emails.